docs/cloud/pouta/launch-vm-from-web-gui.md

```diff
@@ -32,86 +32,142 @@ There are two options for logging in the cPouta's Dashboard: CSC username and pa
 
 For ePouta dashboard, you can only login using the CSC username and password.
 
-### Preparatory steps
+## Preparatory steps
 
-In case you have multiple projects with Pouta access, first select the one you want to use at the menu bar on top left.
+Before creating a Virtual Machine you must do these 3 steps:
+
+1. Select the correct **CSC project**.
+
+1. Create and setup a **SSH key pair**.
+
+1. Setting a **security group** to control the firewall.
 
 Before starting your first virtual machine in cPouta/ePouta, you must first set up a SSH key pair and modify the security settings so that you will be able to connect to your virtual machine.
 
+### Selecting the CSC project
+
+![Pouta project selection](../../img/pouta_project_selection.png){ align=left }
+
+You may have more than one CSC project with access to Pouta. You can check this from [my.csc.fi](https://my.csc.fi){:target="_blank"}, where you will be able to see all the projects you have access and which ones have cPouta (or ePouta) activated as a service.
+
+Back in Poutai's interface, make sure that you select the correct project. There are two condiderations here:
+
+* A project is a sandbox which contains resources like Virtual Machines and networks, and anyone with access to that project will be able to see and administer all these resources. They may not be able to access a Virtual Machine, as this is determinated by the SSH keys configured in the machine, but they will be able to **delete**, **reboot**, ... etc.
+* Projects are used to determinate billing. Make sure that the costs will go the correct billing project.
```

Comment:
perhaps "Make sure that the costs will go *to* the correct billing project"?

Reply:
fixed